js/src/dashboard/all-programs-table-card/index.js

```diff
@@ -44,6 +44,21 @@ const headers = [
 	{ key: 'actions', label: '', required: true },
 ];
 
+function CountryColumn( { countryCodes, countryNameMap } ) {
+	const [ first ] = countryCodes;
+	return (
+		<span>
+			{ countryNameMap[ first ] }
+			{ countryCodes.length >= 2 &&
+				sprintf(
+					// translators: %s: number of countries, with minimum value of 1.
+					__( ' + %s more', 'google-listings-and-ads' ),
```

Comment:
💅 Checking [the repo](https://www.npmjs.com/package/sprintf-js#:~:text=that%20ASCII%20value-,d,-or%20i%20%E2%80%94%20yields)

I'd say this should be %d

```suggestion
					// translators: %d: number of countries, with minimum value of 1.
					__( ' + %d more', 'google-listings-and-ads' ),
```

Reply:
Thanks for catching this! 👍  Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1273/commits/188907e30e8eb401f7a6b044ae639d3e1c67c129.